.bit/responses/2.3-Week 2 Step 3.md

```diff
@@ -7,42 +7,97 @@ step: 3
+
+Then, enter the required information. <br>
+<img width="663" alt="Screen Shot 2021-04-15 at 5 55 41 PM" src="https://user-images.githubusercontent.com/70852990/114943355-dd578600-9e13-11eb-8257-1d9b65f8f5f1.png">
+
 <br>
+Finally, click Create App, and your key should be given!
 
-<details>
-<summary>❓ Where can I find the keys?</summary>
- </br>
+<br><br/>
+</details>
 
-* Navigate to the home page on the Micrsoft Azure portal (https://portal.azure.com/#home)
+## Calling in the Giphy API
+<img src="https://media0.giphy.com/media/833TXYqnMw7Go/200w.webp?cid=ecf05e47z1r1e1ax62zp68eeex57a9bt7vwowretds1ue128&rid=200w.webp&ct=g" height="250">
 
-<img width="1440" alt="Screen Shot 2021-02-04 at 4 00 33 PM" src="https://user-images.githubusercontent.com/28051494/106971033-edac0000-6702-11eb-8243-1b5c2318f76d.png">
+GIPHY Translate converts words and phrases to the perfect GIF or Sticker using GIPHY's special sauce algorithm. For example, the word `beard` gets converted to this masterpiece 
 
-* Click on the resource you need the keys for
+Make sure you edit your `emotional.js` function to call the Giphy API and translate the results from the Face API for a gif that matches the dominant emotion from your emotion data.
 
-<img width="1438" alt="Screen Shot 2021-02-04 at 4 00 49 PM" src="https://user-images.githubusercontent.com/28051494/106971035-ef75c380-6702-11eb-965b-c3ef7b5a7574.png">
+Don't forget to have fun with the API – customize your call using the different parameters. Remember to use the gif and not the sticker endpoint!
 
-* On the left menu bar, locate the Resource Management section and click on "Keys and Endpoint"
+<details>
+<summary>:exclamation: Can I get a more detailed list of steps?</summary>
+  </br>
+  
+1. In the **async function**, create a new constant that will recieve the dominant emotion from your emotion data (i.e. angry, happy). This dominant emotion should be determined before you call on the trigger function.
+2. Using **node-fetch**, call the Giphy **translate endpoint** from the documentation.
+3. Run the constant through the **string** parameter in the endpoint (look as the documentation if this sounds confusing). Add your **Giphy API key**, and set the number of returned gifs to 1 by adding `&limit=1` to the endpoint.
+4. Declare another constant that formats the results of the API call in **json** format.
+5. Make this second constant the **output** of the function in `context.res`.
 
-<img width="1440" alt="Screen Shot 2021-02-04 at 12 26 36 PM" src="https://user-images.githubusercontent.com/28051494/106971042-f43a7780-6702-11eb-9e28-e6b2bc16fa22.png">
-<br>
+  <br><br/>
+  
 </details>
 
-## :camping: To move on, place your two secrets in the repository secrets: `API_ENDPOINT` AND `SUBSCRIPTION_KEY`
+<details>
+<summary>:exclamation: I'm stuck. Can I get some hints?</summary>
+  </br>
+  
+Fill in the gaps:
+  
+```js
+
```

Comment:
Split these hints up into sections, because the student most likely won't need all the hints all at once.